doc/build.sh

```diff
@@ -2,9 +2,27 @@
 
 cd ${0%/*}
 
+weave () {
+    emacs  $1 --batch -l `pwd`/ox-gfm.el \
+	   --eval '(print (org-gfm-export-to-markdown nil nil nil))' --kill ||
+	echo "Cannot export from org $1 to markdown using emacs"
+}
+weave_subtree () {
+    emacs  $1 --batch -l `pwd`/ox-gfm.el \
+	   --eval "(progn (goto-char (+ 1 (org-open-link-from-string \"[[#$2]]\")))
+                          (print (org-gfm-export-to-markdown nil t nil)))" --kill ||
+	echo "Cannot export from org $1 subtree $2 to markdown using emacs"
+}
 
-emacs ../src/std/mime/README.org --batch -f org-gfm-export-to-markdown --kill ||
-    echo "Cannot export from org to markdown using emacs"
+
+
+
+weave ../src/std/mime/README.org
+weave ../src/std/markup/README.org
+weave ../src/std/markup/sxml/README.org
+weave ../src/std/markup/sxml/xml.org
+weave ../src/std/markup/sxml/html/README.org
+weave ../src/std/markup/sxml/tal/README.org
```

Comment:
this script is also used by the netlify build, i hope we didn't break it!
Did you check the output preview?

Reply:
Yup, that's why I added them actually rather than another script somewhere else :)